src/geo/transform.js

```diff
@@ -1661,7 +1687,16 @@ class Transform {
         // Z-axis uses pixel coordinates when globe mode is enabled
         const pixelsPerMeter = this.pixelsPerMeter;
 
-        this._projectionScaler = pixelsPerMeter / (mercatorZfromAltitude(1, this.center.lat) * this.worldSize);
+        if (this.projection.name !== 'globe') {
+            this._projectionScaler = pixelsPerMeter / (mercatorZfromAltitude(1, this.center.lat) * this.worldSize);
+        } else {
+            const refScale = mercatorZfromAltitude(1, 45) * this.worldSize;
```

Comment:
It might be good to document in the code a sentence or two about the thought process that led towards choosing `45` as the reference latitude.

Reply:
Added in [8a21fcc](https://github.com/mapbox/mapbox-gl-js/pull/11951/commits/8a21fcca3e14a0d33152020295181d06bcd244d3). Anything you would change/add here?